AlphaWallet/Market/Coordinators/UniversalLinkCoordinator.swift

```diff
@@ -431,6 +437,11 @@ class UniversalLinkCoordinator: Coordinator {
         return indices.map(String.init).joined(separator: ",")
     }
 
+    private func stringEncodeTokenIds(_ tokenIds: [BigUInt]?) -> String? {
+        guard let tokens = tokenIds else { return nil }
+        return tokens.map({ $0.serialize().hexString }).joined()
```

Comment:
```
return tokens.map({ $0.serialize().hexString }).joined(separator: ",")
```

Reply:
Not necessary here because the paymaster knows each token is 32 bytes

Reply:
Actually, should split by commas, shorter url and more consistent

Reply:
Yes, but also because it's correct? Because we want something like "0x01,0x02" rather than "0x010x02".

Reply:
Was originally thinking of a full padded 32 bytes per token so you could separate but that's long